build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1743,6 +1750,26 @@ message DigestFunction {
     // cryptographic hash function and its collision properties are not strongly guaranteed.
     // See https://github.com/aappleby/smhasher/wiki/MurmurHash3 .
     MURMUR3 = 7;
+
+    // The BLAKE3 digest function, as described in
+    // https://github.com/BLAKE3-team/BLAKE3-specs/blob/master/blake3.pdf ,
```

Comment:
I think we should link to a specific version of this file.

Reply:
The BLAKE3-specs repository isn't tagged, so we'd need to pick an arbitrary commit. I'm fine with that, but at the same time none of the other GitHub URLs in remote_execution.proto point to an exact revision.

Reply:
BLAKE3 is no longer used. Resolving this.